chpl/chpl-service/src/main/java/gov/healthit/chpl/app/TriggerJob.java

```diff
@@ -71,7 +71,8 @@ public static void main(final String[] args) {
                 Scheduler scheduler = sf.getScheduler();
                 for (String group: scheduler.getJobGroupNames()) {
                     for (JobKey jobKey : scheduler.getJobKeys(groupEquals(group))) {
-                        System.out.println("Found job: \"" + jobKey.getName() + "\" in group: \"" + jobKey.getGroup() + "\"");
+                        System.out.println("Found job: \"" + jobKey.getName()
```

Comment:
Since you changed something with this line, can you make it a logged statement instead of system.out?

Reply:
This app is supposed to be called from the command line, and this particular thing is sort of the "help" for using the app. Will LOGGER output to the command line in all cases?

Reply:
Oh.. well it can be configured such that this class outputs to the console instead of a file so that would do it. Understand if you don't want to mess with the configs and leave as is.